### PR TITLE
fix: send ring manual input receiver page error

### LINKED ISSUE #181: Optimization: When I accidentally fill in a number in the receiver，the whole page turns into the error information
image
image

### PR DESCRIPTION
**Pull request by JayJay1024**

### CODE REVIEW

**Review by ZolaxZola — DISMISSED** [review]

File: src/components/account/AssetOverview.tsx [hunk 1]
```diff
@@ -38,9 +39,9 @@ export function AssetOverview({ asset, refresh }: AssetOverviewProps) {
   useEffect(() => {
     let sub$$: Subscription;
 
-    if (recipient && isFunction(api.rpc.payment?.queryInfo)) {
+    if (isFunction(api.rpc.payment?.queryInfo)) {
       if (asset.asset === DarwiniaAsset.ring) {
-        sub$$ = from(api.tx.balances?.transfer(recipient, asset.max).paymentInfo(account)).subscribe((res) => {
+        sub$$ = from(api.tx.balances?.transfer(TEST_ADDR, asset.max).paymentInfo(account)).subscribe((res) => {
```
Comment (sxlwar) [inline src/components/account/AssetOverview.tsx]: TEST_ADDR？

**Review by sxlwar — CHANGES_REQUESTED** [review]

File: src/components/account/AssetOverview.tsx (hunk 1 above)
Comment (JayJay1024) [reply src/components/account/AssetOverview.tsx]: > TEST_ADDR？

yeah, as long as it is a valid address. maybe the name of variable is not so appropriate

Comment (hackfisher) [reply src/components/account/AssetOverview.tsx]: @JayJay1024 Didn't get it, what is this address? 

So strange, please introduce this modification

File: src/components/account/AssetOverview.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { PrettyAmount } from '../widget/PrettyAmount';
 import { BalanceControl } from '../widget/form-control/BalanceControl';
 import { AddressItem } from '../widget/form-control/AddressItem';
 
+const TEST_ADDR = '1ufRSF5gx9Q8hrYoj7KwpzQzDNqLJdbKrFwC6okxa5gtBRd';
```
Comment (hackfisher) [inline src/components/account/AssetOverview.tsx]: What's this?

**Review by hackfisher — CHANGES_REQUESTED** [review]

**Comment by JayJay1024** [comment]
This is used to estimate the gas fee, not the real transaction, as long as this address (TEST_ADDR) is a valid polkadot address.

**Comment by hackfisher** [comment]
Why not use real address for estimation?

If there is no valid address, just do not show the estimation?

cc @ZolaxZola @1022ranji

**Review by ZolaxZola — APPROVED** [review]

**Review by sxlwar — APPROVED** [review]

**Pull request merged**